src/main/java/com/uadudes/book_store/Application.java

```diff
@@ -0,0 +1,32 @@
+package com.uadudes.book_store;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+
+import springfox.documentation.builders.PathSelectors;
+import springfox.documentation.builders.RequestHandlerSelectors;
+import springfox.documentation.spi.DocumentationType;
+import springfox.documentation.spring.web.plugins.Docket;
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
+
+@SpringBootApplication
+@EnableSwagger2
+public class Application
+{
```

Comment:
Oh fuck, I have such format config on work. I'm not paying attention already )
Btw, it's eclipse formatter style :)
Kk, will configure it back